accept_handlers/consts.go

```diff
@@ -5,6 +5,9 @@ const (
 	MIDENVERIFY    = 2
 	RISCZEROVERFIY = 3
 	JOLTVERIFY     = 4
+	PLONKY2VERIFY  = 5
 )
 
 const elfValType = 1
+const commonDataValType = 2
+const verifierDataValType = 3
```

Comment:
Don't hard code commonDataValType & verifierDataValType, let user decide the values and pass with the plonky2 action